File: app-components/src/main/java/org/synsystems/onlypass/components/logging/LoggerTreeModule.java
package org.synsystems.onlypass.components.logging;

import org.synsystems.onlypass.components.AppScope;

import dagger.Module;
import dagger.Provides;

@Module
public class LoggerTreeModule {
  @Provides
  @AppScope
  public LoggerTree provideLoggerTree() {
    return LoggerTree.withoutFiltering();
  }
}

File: app-components/src/main/java/org/synsystems/onlypass/components/logging/LoggerTreeModule.kt
package org.synsystems.onlypass.components.logging

import dagger.Module
import dagger.Provides
import org.synsystems.onlypass.components.AppScope

@Module
class LoggerTreeModule {

  @Provides
  @AppScope
  fun provideLoggerTree() = LoggerTree()
}